source/shared/core_stmt.cpp

```diff
@@ -2321,32 +2299,18 @@ void get_field_as_string( _Inout_ sqlsrv_stmt* stmt, _In_ SQLUSMALLINT field_ind
 
                             field_value_temp = static_cast<char*>( sqlsrv_realloc( field_value_temp, field_len_temp + extra + 1 ));
 
-                            // reset AE stream fetch buffer
-                            if ( reset_ae_stream_cursor( stmt )){
-                                // fetch the original column again with a bigger buffer length
-                                r = stmt->current_results->get_data( field_index + 1, c_type, field_value_temp, field_len_temp + extra,
-                                    &dummy_field_len, false /*handle_warning*/ TSRMLS_CC );
-                                // if field_len_temp was bit enough to hold all data, dummy_field_len contain the actual amount retrieved,
-                                // not SQL_NO_TOTAL
-                                if ( dummy_field_len != SQL_NO_TOTAL )
-                                    field_len_temp = dummy_field_len;
-                                else
-                                    field_len_temp += initial_field_len;
-                            }
-                            else {
-                                field_len_temp -= initial_field_len;
-
-                                // Get the rest of the data.
-                                r = stmt->current_results->get_data( field_index + 1, c_type, field_value_temp + initial_field_len,
-                                    field_len_temp + extra, &dummy_field_len,
-                                    false /*handle_warning*/ TSRMLS_CC );
-                                // the last packet will contain the actual amount retrieved, not SQL_NO_TOTAL
-                                // so we calculate the actual length of the string with that.
-                                if ( dummy_field_len != SQL_NO_TOTAL )
-                                    field_len_temp += dummy_field_len;
-                                else
-                                    field_len_temp += initial_field_len;
-                            }
+                            field_len_temp -= initial_field_len;
+
+                            // Get the rest of the data.
+                            r = stmt->current_results->get_data( field_index + 1, c_type, field_value_temp + initial_field_len,
+                                field_len_temp + extra, &dummy_field_len,
+                                false /*handle_warning*/ TSRMLS_CC );
```

Comment:
This is minor. I know it was like that before, but please fix the indentation. :)